python/tvm/relay/frontend/onnx.py

```diff
@@ -826,6 +826,15 @@ def _impl_v1(cls, inputs, attr, params):
         return out
 
 
+def is_ort_version_greater_than(ver):
+    import onnxruntime as ort
+
+    v11, v12, v13 = tuple(int(v) for v in ort.__version__.split("."))
```

Comment:
probably you can try to use get_converter here

Reply:
What do you mean exactly?

Frankly speaking, at the end, probably there should be no need for this function at all, due to the fact that for opset 11 of convTranspose operator there is only one valid implementation.
On the other hand, that implementation should be independent from the version of ONNXRuntime version.

But, due to the change in the interpretation of autopad parameters in ONNXRuntime versions 1.12.1 and 1.13.1,the same models produce different results in different ONNXRuntime versions, and since there are tests in TVM that compares TVM results with runtime results, I added this additional check in order to make that TVM results be equal to ONNXRuntime in both cases. 

There is even a warning when trying to run onnx model with convTranspoose operator with version 1.12.1
`[W:onnxruntime:Default, conv_transpose.h:32 ConvTranspose] The existing bug in the padding distribution for auto_pad type SAME_UPPER/SAME_LOWER will be fixed in next ORT 1.13 release and hence the results of ConvTranspose operator using the above auto_pad type(s) will be different.`